### PR TITLE
Add functions to remove objects

### PR DESCRIPTION
**Pull request by c0dehard**
- Customize cmd/{server,storage,network}.go to match refactor
- Add gscloud server rm
- Add gscloud storage rm
- Add gscloud network rm
- Append storage_test.go

Fixes GD-2685

### CODE REVIEW

**Comment by nvthongswansea** [comment]
Since you've changed the signature of `produceNetworkCmdRunFunc()` function, you need to change the input of that function in its unit test as well.

**Comment by c0dehard** [comment]
Oh, i see! Totally forgot thanks 👋

File: cmd/server.go [hunk 1]
```diff
@@ -133,8 +143,16 @@ func initServerCmd() {
 		Args:  cobra.ExactArgs(1),
 		Run:   produceServerCmdRunFunc(client, serverStopAction),
 	}
-	serverOffCmd.PersistentFlags().BoolVarP(&forceFlag, "force", "f", false, "Force shutdown (no ACPI)")
+	var removeCmd = &cobra.Command{
+		Use:     "rm [flags] [ID]",
+		Aliases: []string{"remove"},
+		Short:   "Remove Server",
+		Long:    `Remove an existing Server.`,
```
Comment (bkircher) [inline cmd/server.go:150]: Both "Server" please lowercase.

File: cmd/storage.go [hunk 2]
```diff
@@ -65,17 +86,24 @@ func initStorageCmd() {
...
 		Short:   "List storages",
 		Long:    `List storage objects.`,
-		Run:     produceStorageCmdRunFunc(client),
+		Run:     produceStorageCmdRunFunc(client, storageListAction),
+	}
+	var removeCmd = &cobra.Command{
+		Use:     "rm [flags] [ID]",
+		Aliases: []string{"remove"},
+		Short:   "Remove Storage",
+		Long:    `Remove an existing Storage.`,
```
Comment (bkircher) [inline cmd/storage.go:102]: Same here. Both "Storage" please lowercase.

File: cmd/storage_test.go [hunk 3]
```diff
@@ -92,3 +96,22 @@ func Test_StorageCmdOutput(t *testing.T) {
...
+	os.Stdout = w
+	mockClient := mockStorageGetter{}
+	cmd := produceStorageCmdRunFunc(mockClient, storageDeleteAction)
+	cmd(new(cobra.Command), []string{"rm", mockStorage.Properties.ObjectUUID})
+	// close the write side of the pipe so that we can start reading from
+	// the read side of the pipe
+	w.Close()
+	// Read the standard output's result from the read side of the pipe
+	out, _ := ioutil.ReadAll(r)
+	assert.Equal(t, err.expectedOutput, string(out))
```
Comment (bkircher) [inline cmd/storage_test.go:116]: Can you remove the source comments here? ode is pretty self-explanatory. Only here please.

File: cmd/storage.go (hunk 2 above)
Comment (c0dehard) [reply cmd/storage.go]: Oops, of course 😃

File: cmd/server.go (hunk 1 above)
Comment (c0dehard) [reply cmd/server.go]: Done 👍

File: cmd/storage_test.go (hunk 3 above)
Comment (c0dehard) [reply cmd/storage_test.go:114]: As far I understood, only removed the following comment: `// Read the standard output's result from the read side of the pipe` ?

**Comment by bkircher** [comment]
LGTM. Works as advertised :+1:

**Pull request merged**